python/tk_multi_loader/delegate_publish_list.py

```diff
@@ -165,7 +165,11 @@ def _format_publish(self, model_index, widget):
         # Publish Name Version 002
         sg_data = shotgun_model.get_sg_data(model_index)
         main_text = "<b>%s</b>" % (sg_data.get("name") or "Unnamed")
-        main_text += " Version %03d" % sg_data.get("version_number")        
+
+        version = sg_data.get("version_number")
+        vers_str = "%03d" % version if version else "N/A"
```

Comment:
`if version is not None` would be better. It is possible to publish version 0 (I've seen it before!)